l10n_it_fatturapa/bindings/binding.py

```diff
@@ -26,9 +35,7 @@
 # inside class definitions where property names may conflict.
 _module_typeBindings = pyxb.utils.utility.Object()
 
-# Import bindings for namespaces imported into schema
-import pyxb.binding.datatypes
-import _ds as _ImportedBinding__ds
+import _ds as _ImportedBinding__ds  # pylint: disable=relative-import
```

Comment:
Meglio seguire il consiglio di Travis invece di sopprimere `lint`: `Relative import '_ds', should be 'l10n_it_fatturapa.bindings._ds'`

Reply:
fatto

Reply:
Grazie! Nel diff però c'è ancora qualche `# pylint: ...` (tipo https://github.com/OCA/l10n-italy/pull/1351/files#diff-16dbec4eeba59d61c6c9834943d05606R201), ma a codice l'hai tolto correttamente; credo manchi di rigenerare il `.diff`

Reply:
Grazie, fatto

Reply:
@eLBati non capisco cosa stia succedendo con questo diff ma ci sono ancora dei `# pylint: ...` non presenti a codice tipo https://github.com/OCA/l10n-italy/pull/1351/files#diff-16dbec4eeba59d61c6c9834943d05606R306.
Forse per semplificarci la vita si potrebbe evitare il diff e nel README scrivere qualcosa tipo _Assicurarsi che il codice autogenerato rispetti gli standard del repository_

Reply:
@SimoRubi ho scoperto che c'era anche il diff del diff...
Ora dovrebbe essere OK